_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -386,13 +386,16 @@ public static function get_plans( $request ) {
 		$data = $use_cache ? get_transient( 'jetpack_plans' ) : false;
 
 		if ( false === $data ) {
-			$path = '/plans';
+			$path = '/plans?_locale=' . get_user_locale();
 			// passing along from client to help geolocate currency
-			$ip = $_SERVER['HTTP_X_FORWARDED_FOR']; // if we already have an list of forwarded ips, then just use that
-			if ( empty( $ip ) ) {
+			if ( isset( $_SERVER['HTTP_X_FORWARDED_FOR'] ) ) {
+				$ip = $_SERVER['HTTP_X_FORWARDED_FOR']; // if we
+			}
+			//already have an list of forwarded ips, then just use that
+			if ( empty( $ip ) && isset( $_SERVER['HTTP_CLIENT_IP'] )) {
 				$ip = $_SERVER['HTTP_CLIENT_IP']; // another popular one for proxy servers
 			}
-			if ( empty( $ip ) ) {
+			if ( empty( $ip ) && isset( $_SERVER['REMOTE_ADDR'] ) ) {
```

Comment:
Is there a value `$ip` could default to just in case none of these exist? (e.g. when they're using some other proxy header not listed here).